osgbuild/importer.py

```diff
@@ -60,13 +60,19 @@ def __init__(self, msg):
 
 def download_srpm(url, output=None):
     """Download an srpm from url. Return the filename."""
-    # TODO: This should probably use urllib2
     if output is None:
         output = os.path.basename(url)
     cmd = ["wget", "-q", url, "-O", output]
     utils.checked_call(cmd)
+    verifyRPM(output)
     return output
 
+def verifyRPM(srpm):
+    """ Verify that srpm is indeed an RPM """
+    cmd = ["rpm", "-qp", srpm]
```

Comment:
Could you add "--nomanifest" to the argument list? It will prevent rpm -qp from hilariously misinterpreting certain kinds of files.

Reply:
done and tested thank you.